packages/editor-ui/src/components/MainHeader/CollaborationPane.vue

```diff
@@ -1,22 +1,28 @@
 <script setup lang="ts">
-import { COLLABORATION_HEARTBEAT_INTERVAL } from '@/constants';
 import { useUsersStore } from '@/stores/users.store';
 import { useWorkflowsStore } from '@/stores/workflows.store';
 import { useCollaborationStore } from '@/stores/collaboration.store';
 import { onBeforeUnmount } from 'vue';
 import { onMounted } from 'vue';
 import { computed, ref } from 'vue';
+import { TIME } from '@/constants';
 
 const collaborationStore = useCollaborationStore();
 const usersStore = useUsersStore();
 const workflowsStore = useWorkflowsStore();
 
-const heartbeatInterval = COLLABORATION_HEARTBEAT_INTERVAL;
-const heartbeatTimer = ref(null as null | NodeJS.Timeout);
+const HEARTBEAT_INTERVAL = 5 * TIME.MINUTE;
+const heartbeatTimer = ref(null as null | number);
```

Comment:
`ref` supports a way to define the type without using a type cast

```suggestion
const heartbeatTimer = ref<number | null>(null);
```